checkov/terraform/checks/resource/base_resource_negative_value_check.py

```diff
@@ -0,0 +1,38 @@
+from abc import abstractmethod
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class BaseResourceNegativeValueCheck(BaseResourceCheck):
+    def __init__(self, name, id, categories, supported_resources):
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        raise NotImplementedError()
+
+    @abstractmethod
+    def get_inspected_key(self):
+        """
+        :return: JSONPath syntax path of the checked attribute
+        """
+        raise NotImplementedError()
+
+    def get_bad_values(self):
```

Comment:
I would rename from `bad` to `forbidden`

Reply:
Done.